### PR TITLE
2022 data update

### PR DESCRIPTION
**Pull request by grc54**
Update 2022 data, create the data subset, update the documentation

### CODE REVIEW

File: inst/sampling.R
Comment (kevinwang09) [inline inst/sampling.R:1]: @grc54, please use set.seed() to guarantee reproducibility. 
@dicook, I think we should do this for every year, not just the 2022 data. However, this means that the current subset data in the package may change, so we won't be able to guarantee 'backwards compatibility'. Thoughts?

**Review by kevinwang09 — CHANGES_REQUESTED** [review]

**Review by grc54 — COMMENTED** [review]
Add seed for reproducibility of dataset

File: inst/sampling.R
Comment (kevinwang09) [inline inst/sampling.R:1]: Please do the sampling process for every year, and update the small data files too.

**Review by kevinwang09 — CHANGES_REQUESTED** [review]

File: inst/sampling.R
Comment (grc54) [reply inst/sampling.R:1]: completed and updated the data files

**Pull request merged**